tests/json_gen/generator.py

```diff
@@ -96,11 +166,30 @@ def _gen_json(self, schema: dict):
         try:
             # should be inside an object now otherwise there should be a ref.
             ref = schema.get(u"$ref")
-            json_type = schema.get(u"type", "object")
             if ref is not None:
                 impostor = self._ref(ref)
             else:
-                impostor = getattr(self, f"_{json_type}")(schema)
+                temp_schema = deepcopy(schema)
+                all_of = temp_schema.get('allOf')
+                if all_of is not None:
+                    for subschema in all_of:
+                        _update(temp_schema, subschema)
+                any_of = temp_schema.get('anyOf')
+                if any_of is not None:
+                    for subschema in random.sample(any_of, random.randint(1, len(any_of) - 1)):
+                        _update(temp_schema, subschema)
```

Comment:
Mention that there can be contradictions in any off that produce an invalid schema.

Reply:
only chose one, also added a comment